x-pack/plugins/security_solution/public/common/components/cell_actions/index.test.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { SecurityCellActionsTrigger } from '../../../actions/constants';
+import { CellActionsMode, SecurityCellActions } from '.';
+import { CellActions } from '@kbn/cell-actions';
+
+const MockCellActions = CellActions as jest.Mocked<typeof CellActions>;
+jest.mock('@kbn/cell-actions', () => ({
+  ...jest.requireActual('@kbn/cell-actions'),
+  CellActions: jest.fn(() => <div data-test-subj="cell-actions-component" />),
+}));
+
+const mockGetFieldSpec = jest.fn((_: string) => ({ someFieldSpec: 'theFieldSpec' }));
+const mockUseGetFieldSpec = jest.fn((_: unknown) => mockGetFieldSpec);
+jest.mock('../../hooks/use_get_field_spec', () => ({
+  useGetFieldSpec: (param: unknown) => mockUseGetFieldSpec(param),
+}));
+const mockDataViewId = 'security-default-dataview-id';
+const mockUseDataViewId = jest.fn((_: unknown) => mockDataViewId);
+jest.mock('../../hooks/use_data_view_id', () => ({
+  useDataViewId: (param: unknown) => mockUseDataViewId(param),
+}));
+
+const defaultProps = {
+  triggerId: SecurityCellActionsTrigger.DEFAULT,
+  mode: CellActionsMode.INLINE,
+};
+const mockData = [{ field: 'fieldName', value: 'fieldValue' }];
+const mockMetadata = { someMetadata: 'value' };
+
+describe('SecurityCellActions', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should render children when fieldData length is 0', () => {
```

Comment:
NIT: As both conditions render `children`, highlight the main difference would be clearer to me:
```suggestion
  it('should not render CellActions component when fieldData length is 0', () => {
```